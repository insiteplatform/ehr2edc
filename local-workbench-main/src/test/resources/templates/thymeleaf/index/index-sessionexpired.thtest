%TEMPLATE_MODE HTML5
# ------------------------------------------
%INPUT (classpath:templates/thymeleaf/index.html)
%FRAGMENT div[@id='sessionExpired']
# ------------------------------------------
%OUTPUT (classpath:templates/thymeleaf/index/expected/index-sessionexpired.html)
