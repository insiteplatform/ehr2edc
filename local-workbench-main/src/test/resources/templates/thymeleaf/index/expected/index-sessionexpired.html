<div id="sessionExpired" class="modal hide">
    <aside class="modal is-active">
        <h3 class="border-bottom">Session Expired</h3>
        <p class="margin-tb">Please <strong><a href="/testing/auth/login" target="_blank">sign in</a></strong> again first.
        </p>
        <p>A new window will open. Afterwards, return to this window and continue.</p>
        <a href="/testing/auth/login" class="button primary no-unload" target="_blank">Sign in again</a>
    </aside>
</div>